token/js/package.json

```diff
@@ -1,11 +1,28 @@
 {
-  "name": "spl-token",
+  "name": "@solana/spl-token",
   "version": "0.0.1",
-  "description": "",
+  "description": "SPL Token Javascript API",
+  "license": "MIT",
+  "author": "Solana Maintainers <maintainers@solana.com>",
+  "homepage": "https://solana.com/",
   "repository": {
     "type": "git",
     "url": "https://github.com/solana-labs/solana-program-library"
   },
+  "bugs": {
+    "url": "https://github.com/solana-labs/solana-program-library/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.esm.js",
+  "types": "lib/index.d.ts",
```

Comment:
You'll need to setup a build command to create these files. Looks like you'll also want a `token/js/index.js` file to export everything that you've added types for in `module.d.ts`